libs/config/common/src/lib/endpointLoader.ts

```diff
@@ -18,7 +18,7 @@ export type DynamicEndpoints = { [key in Endpoint]: string | undefined };
 export type LoadedEndpoints = { [key in Endpoint]: string };
 
 export class EndpointLoader {
-  public useDevRuns = false;
+  public useDevRuns = true;
```

Comment:
don't forget, this logic should be grabbed from the environment.env configuration variable if possible. ... maybe next PR.

Reply:
Correct.